src/Init/Data/Char/Basic.lean

```diff
@@ -5,6 +5,7 @@ Author: Leonardo de Moura
 -/
 prelude
 import Init.Data.UInt.Basic
+import Init.Data.Char.UnicodeVersion
```

Comment:
Perhaps better to import this at `Init/Data/Char`, as it is not actually needed here.